pom.xml

```diff
@@ -126,6 +126,9 @@
           <groupId>com.spotify</groupId>
           <artifactId>docker-maven-plugin</artifactId>
           <version>0.3.2</version>
+          <configuration>
```

Comment:
Good catch...
I am using Docker Mac beta early access, and this is a new setting for this
non standard docker.
I need to remove my specific setting until we standardize with this Docker
Mac version...


On Thu, Apr 28, 2016 at 4:31 PM, Arie <notifications@github.com> wrote:

> In pom.xml
> <https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/pull/227#discussion_r61518861>
> :
>
> > @@ -126,6 +126,9 @@
> >            <groupId>com.spotify</groupId>
> >            <artifactId>docker-maven-plugin</artifactId>
> >            <version>0.3.2</version>
> > +          <configuration>
>
> what is that for?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/pull/227/files/1aeae8cec7dcfe214643f43c16376aa454b9e26a#r61518861>
>